ci-bench/src/rustls_facade.rs

```diff
@@ -0,0 +1,60 @@
+use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+use rustls::{ConnectionCommon, SideData};
+use std::io::{Read, Write};
+
+/// Sends data to the other side of the connection through the provided writer
+///
+/// Prepends a header consisting of the length of the TLS data (encoded as a big-endian u32), so the
+/// other side can know how many bytes to expect.
+pub fn send<T: SideData>(
```

Comment:
Can we reuse library elements like `complete_io()` instead?

Reply:
Thanks for the tip, I didn't know about `complete_io`. I have pushed changes to start using it.